src/fetchRequests.js

```diff
@@ -21,6 +21,27 @@ const fetchRequests = {
       .then(response => response.json())
       .then(data => data.trips)
       .catch(error => console.log(error))
+  },
+  postNewTrip(trip) {
+    fetch('https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/trips/trips', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        "id": trip.id,
+        "userID": trip.userID,
+        "destinationID": trip.destinationID,
+        "travelers": trip.travelers,
+        "date": trip.date,
+        "duration": trip.duration,
+        "status": trip.status,
+        "suggestedActivities": trip.suggestedActivities
+      })
+    })
+      .then(response => response.json())
+      .then(data => console.log(data))
```

Comment:
Is there anything you can think of that you can do with this data rather than just a `console log`? It's super powerful to have access to the fulfilled `Promise` and do stuff with it.